packages/interactivity/src/index.js

```diff
@@ -7,8 +7,9 @@ import { init } from './router';
 export { store } from './store';
 export { directive, getContext, getElement } from './hooks';
 export { navigate, prefetch } from './router';
-export { h as createElement } from 'preact';
-export { useEffect, useContext, useMemo } from 'preact/hooks';
+export { useWatch, useInit } from './utils';
+export { h as createElement, cloneElement } from 'preact';
+export { useEffect, useLayoutEffect, useContext, useMemo } from 'preact/hooks';
```

Comment:
Oh wait, can you export the scoped versions of the preact hooks as well?

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/57805/commits/3f34051d3b80b040ed2f01d33e5f2fabe65df1d5.